src/plugins/Trader/UI/Linking.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Link } from '@material-ui/core'
+
+export interface LinkingProps {
+    href?: string
+    children?: React.ReactNode
+}
+
+export function Linking(props: LinkingProps) {
+    return (
+        <>
+            {props.href ? (
+                <Link color="primary" target="_blank" rel="noopener noreferrer" href={props.href}>
```

Comment:
This component seems familiar, maybe we need to DRY on this component.

Reply:
Which one?

Reply:
Maybe you can search for `target="_blank"` in the project

Reply:
Failed!

Reply:
Ok I mean `rel="noopener noreferrer"`, there are 4 results in the project (and maybe more cases we forgot that)

Reply:
This component creates a `<Link />` if needed, other use cases in this project use `<Link />` directly.